sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/pom.xml

```diff
@@ -18,6 +18,7 @@
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
   <version>1.21.0-beta.2</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;current} -->
+  <version>1.21.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;current} -->
```

Comment:
There are two version tags here

Reply:
Ahh I must've missed this when resolving the cherry-pick issues. I'll fix this.